File: automacao/programacao-frc/exemplo-speedgroup.cpp
#include <frc/Joystick.h>
#include <frc/SpeedControllerGroup.h>
#include <frc/Spark.h>

class Robot : public frc::TimedRobot
{
	frc::Spark m_motor1{1};
	frc::Spark m_motor2{2};

	frc::SpeedControllerGroup m_motores{m_motor1, m_motor2};

public:
	void TeleopPeriodic()
	{
		if (m_stick.GetRawButton(0))
		{
			m_motores.Set(0.4);
		}
		else
		{
			m_motores.Set(0.0);
		}
	}
};

#ifndef RUNNING_FRC_TESTS
int main()
{
	return frc::StartRobot<Robot>();
}
#endif

File: automacao/programacao-frc/exemplo-speedgroup.java
import edu.wpi.first.wpilibj.Joystick;
import edu.wpi.first.wpilibj.Spark;
import edu.wpi.first.wpilibj.SpeedControllerGroup;

public class Robot extends TimedRobot {
	private final Joystick m_stick = new Joystick(0);
	private final Spark m_motor1 = new Spark(1);
	private final Spark m_motor2 = new Spark(2);
	SpeedControllerGroup m_motores = new SpeedControllerGroup(m_motor1, m_motor2);

	@Override
	public void teleopPeriodic() {
		if (m_stick.getRawButton(0)) {
			m_motores.set(0.4);
		} else {
			m_motores.set(0.0);
		}
	}
}
